Library/Homebrew/unversioned_cask_checker.rb

```diff
@@ -88,8 +98,9 @@ def all_versions
 
         installer.extract_primary_container(to: dir)
 
-        info_plist_paths = apps.concat(qlplugins).flat_map do |artifact|
-          top_level_info_plists(Pathname.glob(dir/"**"/artifact.source.basename/"Contents"/"Info.plist")).sort
+        info_plist_paths = apps.concat(qlplugins, installers).flat_map do |artifact|
+          source = artifact.is_a?(Cask::Artifact::Installer) ? artifact.path : artifact.source.basename
```

Comment:
Why do installers need to use `path` here?

Reply:
Most Cask::Artifact inherit `#source` from #Relocatable. However, installer does is not a child of Relocatable so does have `#source` but has `#path` instead.

Reply:
Yeah, same in the signing audit